packages/cli/src/middlewares/cors.ts

```diff
@@ -8,7 +8,7 @@ export const corsMiddleware: RequestHandler = (req, res, next) => {
 		res.header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE');
 		res.header(
 			'Access-Control-Allow-Headers',
-			'Origin, X-Requested-With, Content-Type, Accept, sessionid',
+			'Origin, X-Requested-With, Content-Type, Accept, push-ref',
```

Comment:
```suggestion
			'Origin, X-Requested-With, Content-Type, Accept, push_ref',
```

correct me if I'm wrong but I think the header is part of the `/rest/settings` and it's `push_ref` not `push-ref`